### ISSUE TITLE
How should Math be represented in epub?

### ISSUE THREAD

**Issue by RachelComerford** [w3c/publ-cg#29]
Need to fill in best practices documentation for math.

- Visual appearance, especially in education and journals is important.
- Must be accessible

**Comment by RachelComerford** [comment]
MathML is supported in some reading systems, even those that are based on platforms that don't support MathML, but other systems will not accept files that contain MathML. This means that publishers who supply one epub for all retailers cannot use MathML. Supplying an altimage within mathML is acceptable. I just don't want my file to be rejected because it contains MathML.

**Comment by deborahgu** [comment]
1. Can this best practice include a (very simple) flow chart? Eg. If you make one epub for all reading systems, do this. If you can provide alternate renditions, do this. Etc?
1. Will the best practice take the reality of AT-support of MathML, <mathml alt=> and mathjax into account, or does the best practices say "follow the spec and AT needs to get their act together"?

**Comment by TzviyaSiegman** [comment]
@GeorgeKerscher showed me a work in progress from DIAGRAM and DAISY that would be helpful.
